packages/@o3r/dev-tools/src/utils/jest.helpers.ts

```diff
@@ -54,14 +54,15 @@ const findParentPackageJson = (directory: string, rootDir?: string): string | un
  * @deprecated Please use `pathsToModuleNameMapper` from `ts-jest`, will be removed in Otter v12.
  * @param rootDir Root directory of the jest project
  * @param testingTsconfigPath Path to the tsconfig.json used for test mapping files
- * @returns
+ * @returns {Record<string, any>} List of modules mapping
```

Comment:
```suggestion
```
not needed in typescript files as we already have the typings and the description of the function